### PR TITLE
added support for not in operator

### PR DESCRIPTION
**Pull request by skjindal93**
## Description
Added support for NOT IN operator

### CODE REVIEW

**Review by aaron-steinfeld — DISMISSED** [review]

**Comment by skjindal93** [comment]
Weird. Even after the snyk failure, the merge option was still on!

**Comment by aaron-steinfeld** [comment]
> Weird. Even after the snyk failure, the merge option was still on!

snyk isn't required, although probably should be

File: hypertrace-core-graphql-platform/build.gradle.kts [hunk 1]
```diff
@@ -30,11 +30,13 @@ dependencies {
 
     runtime("org.apache.logging.log4j:log4j-slf4j-impl:2.14.0")
     runtime("io.grpc:grpc-netty:1.35.0")
-    runtime("io.netty:netty-codec-http2:4.1.53.Final") {
-      because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-1020439")
-    }
-    runtime("io.netty:netty-handler-proxy:4.1.53.Final") {
-      because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-1020439s")
+    constraints {
```
Comment (aaron-steinfeld) [inline hypertrace-core-graphql-platform/build.gradle.kts:33]: you're already in a constraints block here

Comment (skjindal93) [reply hypertrace-core-graphql-platform/build.gradle.kts:33]: oh 🤣

Comment (aaron-steinfeld) [reply hypertrace-core-graphql-platform/build.gradle.kts]: also, new snyk id - https://snyk.io/vuln/SNYK-JAVA-IONETTY-1070799

**Review by aaron-steinfeld — APPROVED** [review]

**Pull request merged**